.azure-pipelines/stages.yml

```diff
@@ -143,7 +143,7 @@ stages:
       authGCP: $(GcpServiceAccountKey)
       authGithub: $(GitHubPublicRepoOnlyAccessToken)
       authGPGPassphrase: $(MaintainerGPGKeyPassphrase)
-      authGPGKey: $(MaintainerGPGKey)
+      authGPGKey: $(MaintainerGPGKeySecureFileDownloadPath)
```

Comment:
rather than the cloud variable name, I think the name of local variables like `authGPGKey`, `authGPGPath` need to be renamed.
review all these variables, I think the cloud variables' names just fine.

Reply:
there seems to be some (!) confusion here

this is not disambiguation those - this is disambiguating the name of the task from var that holds the path to the secure file

Reply:
> there seems to be some (!) confusion here
> 
> this is not disambiguating those - this is disambiguating the name of the task from var that holds the path to the secure file

I see.

Reply:
the difference is `MaintainerGPGKeyPassphrase` is a string

whereas `MaintainerGPGKeySecureFileDownloadPath` is a string that references a different type of object (ie `SecureFile`)